packages/legend-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -98,9 +98,18 @@ const PropertyBasicEditor = observer(
       property.owner instanceof Class && property.owner !== _class;
     const isPropertyFromAssociation = property.owner instanceof Association;
     const isIndirectProperty = isInheritedProperty || isPropertyFromAssociation;
+    const checkDuplicateProperty = (val: string): boolean => {
+      let count = 0;
+      _class.properties.forEach((property) => property.name === val && count++);
```

Comment:
also, why don't we do `_class.properties.filter((prop) => prop.name === val.name).length >= 2`? Could we do this for other places?